### PR TITLE
[template/angular] Fix `/graphql` page for all modes

### PR DESCRIPTION
**Pull request by sc-addypathania**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [x] Changelog updated
- [ ] Upgrade guide entry added

## Description / Motivation
Follow up to Angular 17 upgrade.
Bug--> /graphql pages not working.
After reverting apollo-angular dependency everything works as expected.
Graphql dependencies will be updated with Angular 18 since some of them are not compatible with 17.

## Testing Details

- [ ] Unit Test Added
- [x] Manual Test/Other (Please elaborate)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by art-alexeyenko — APPROVED** [review]
👍

File: packages/create-sitecore-jss/src/templates/angular/package.json [hunk 1]
```diff
@@ -61,7 +61,7 @@
     "@ngx-translate/core": "~15.0.0",
     "@ngx-translate/http-loader": "~8.0.0",
     "@sitecore-jss/sitecore-jss-angular": "~22.0.0",
-    "apollo-angular": "~7.0.2",
+    "apollo-angular": "~5.0.2",
```
Comment (illiakovalenko) [inline packages/create-sitecore-jss/src/templates/angular/package.json:64]: @sc-addypathania @art-alexeyenko "apollo-angular" 5.0.2 is supposed to be used with Angular 16, this change is not correct. You need to upgrade at least to version 6

Comment (sc-addypathania) [reply packages/create-sitecore-jss/src/templates/angular/package.json:64]: Hmm you're right. I updated the dependency within the already scaffolded app then it worked fine but new app fails.
Issued another PR.

Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/templates/angular/package.json:64]: @sc-addypathania Thanks!

**Pull request merged**